utils/github_reporting.py

```diff
@@ -52,6 +52,9 @@ def format_github_link(link):
 def format_results(results, jobs, reference_jobs=None, instability_analysis=None, github_links=None):
 
     output_string = "{}\n" \
+                    "> [!WARNING]\n" \
+                    "> Failed tests may be incomplete if they failed to " \
+                    "upload, always check the complete test suite link.\n\n" \
```

Comment:
What is missing is listing upload failures - which this script _does_ know about...

Reply:
Specifically https://github.com/QubesOS/openqa-tests-qubesos/blob/main/utils/lib/openqa_api.py#L229 and https://github.com/QubesOS/openqa-tests-qubesos/blob/main/utils/lib/openqa_api.py#L622-L629 - failure in "system_tests" job should be reported as upload failure if there are no detailed results in that job.